src/benchmarklib/benchmark_runner.cpp

```diff
@@ -410,7 +410,7 @@ cxxopts::Options BenchmarkRunner::get_basic_cli_options(const std::string& bench
     ("visualize", "Create a visualization image of one LQP and PQP for each query, do not properly run the benchmark", cxxopts::value<bool>()->default_value("false")) // NOLINT
     ("verify", "Verify each query by comparing it with the SQLite result", cxxopts::value<bool>()->default_value("false")) // NOLINT
     ("cache_binary_tables", "Cache tables as binary files for faster loading on subsequent runs", cxxopts::value<bool>()->default_value("false")) // NOLINT
-    ("sql_metrics", "Track SQL metrics (parse time etc.) for each SQL query", cxxopts::value<bool>()->default_value("false")); // NOLINT
+    ("sql_metrics", "Track SQL metrics (parse time etc.) for each SQL query (needs output file to be set)", cxxopts::value<bool>()->default_value("false")); // NOLINT
```

Comment:
`Track SQL metrics (parse time etc.) for each SQL query and adds it to the output JSON`?

Reply:
How about `Track SQL metrics (parse time etc.) for each SQL query and adds it to the output JSON (if set)` to make clear that this parameter depends on `-o`?

Reply:
`Track SQL metrics (parse time etc.) and add it to the output JSON (see -o)`?